src/main/main.ts

```diff
@@ -33,7 +33,7 @@ export async function onReady() {
   const { setupMenu } = await import('./menu');
   const { setupFileListeners } = await import('./files');
 
-  setupMenu();
+  setupMenu({});
```

Comment:
We could probably make the options object optional here, and make `setupMenu()` behaviour the same as `setupMenu({})`.